packages/@aws-cdk/aws-appmesh/lib/virtual-gateway-listener.ts

```diff
@@ -106,24 +153,33 @@ class HttpGatewayListener extends VirtualGatewayListener {
    */
   protected protocol: Protocol = Protocol.HTTP;
 
-  constructor(options: HttpGatewayListenerOptions = {}) {
+  /**
+   * Represents the listener certificate
+   *
+   * @default - none
+   */
+  protected readonly tlsCertificate?: TlsCertificate;
+
+  constructor(props: HttpGatewayListenerOptions = {}) {
```

Comment:
This rename is incorrect. It should still be called `options`. `props` are reserved for properties of Constructs, which `VirtualGatewayListener` is not.